src/MSBuildTaskHost/MSBuildTaskHost.csproj

```diff
@@ -45,6 +45,9 @@
     <Compile Include="..\Framework\BuildEngineResult.cs">
       <Link>BuildEngineResult.cs</Link>
     </Compile>
+    <Compile Include="..\Framework\TaskExecutionContext.cs">
+      <Link>BuildEngineResult.cs</Link>
```

Comment:
Wrong link?

Reply:
link fixed, still how come it compiles...?